Library/Formula/sbcl.rb

```diff
@@ -97,6 +98,11 @@ def install
 
     ENV["INSTALL_ROOT"] = prefix
     system "sh", "install.sh"
+
+    if build.with? "sources"
+      system "cp", "-R", "contrib", prefix
+      system "cp", "-R", "src", prefix
```

Comment:
The audit failure is for these lines:
```
        Error: 2 problems in 1 formula
sbcl:
 * Use the `cp` Ruby method instead of `system "cp"`
 * Use the `cp` Ruby method instead of `system "cp"`